assets/components/openshift-router/deployment.yaml

```diff
@@ -7,6 +7,7 @@ spec:
     metadata:
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+        openshift.io/required-scc: restricted
```

Comment:
Can you tell why it isn't included in the upstream file (https://github.com/openshift/cluster-ingress-operator/blob/master/pkg/manifests/assets/router/deployment.yaml#L10-L11)? It is set during runtime by the operator?

This file is copied when microshift updates its dependencies, so this change might be overwritten in few days if we merge this. If this is runtime addition, then we also need a change in rebase.sh script ([see this](https://github.com/openshift/microshift/blob/main/scripts/auto-rebase/rebase.sh#L684-L751)).

I believe this should do the job:
```
yq '.spec.template.metadata.annotations += {"openshift.io/required-scc": "restricted"}' ./assets/components/openshift-router/deployment.yaml
```

Reply:
Yes, the operator is setting it dynamically based on whether it needs host networking or not here:  https://github.com/openshift/cluster-ingress-operator/blob/7dc12cf28557f8c80460435649cfc393761e0285/pkg/operator/controller/ingress/deployment.go#L1212-L1215

I don't see where Microshift has the same host networking config, so just setting it to `restricted` here.

I was not aware of the rebase script - made the change there. Thanks!